source/tutorials/apache-hadoop.rst

```diff
@@ -3,33 +3,41 @@
 Apache\* Hadoop\*
 #################
 
-This tutorial walks you through the process of installing, configuring, and
-running Apache Hadoop on |CL-ATTR|. The Apache Hadoop software library is a
-framework for distributed processing of large data sets across clusters of
-computers using simple programming models. It is designed to scale up from
-single servers to thousands of machines, with each machine offering local
-computation and storage.
+This tutorial explains the process of installing, configuring, and
+running Apache Hadoop on |CL-ATTR|.
+
+.. contents::
+   :local:
+   :depth: 1
+
+Description
+***********
+
+The Apache Hadoop software library is a framework for distributed processing
+of large data sets across clusters of computers using simple programming
+models. It is designed to scale up from single servers to thousands of
+machines, with each machine offering local computation and storage.
 
 Prerequisites
 *************
 
-Before following this tutorial, you should follow the
+Before following this tutorial, follow the
 :ref:`bare-metal-install-desktop` to ensure you have installed |CL|.
 
-Before you install any new packages, update |CL| with the following command:
+Before installing any new packages, update |CL| with the following command:
 
 .. code-block:: bash
 
    sudo swupd update
 
-For the purposes of this tutorial, we will install Hadoop in a single machine
+For this tutorial, you will install Hadoop in a single machine
```

Comment:
Please move this sentence to the first in the para under "Description."

Reply:
Done.